OpenRA.Mods.Common/Widgets/Logic/GlobalChatLogic.cs

```diff
@@ -38,9 +38,8 @@ public GlobalChatLogic(Widget widget)
 			inputBox.IsDisabled = () => Game.GlobalChat.ConnectionStatus != ChatConnectionStatus.Joined;
 			inputBox.OnEnterKey = EnterPressed;
 
-			// Set a random default nick
-			if (Game.Settings.Chat.Nickname == new ChatSettings().Nickname)
-				Game.Settings.Chat.Nickname += Game.CosmeticRandom.Next(100, 999);
+			if (string.IsNullOrEmpty(Game.Settings.Chat.Nickname))
```

Comment:
`Game.Settings.Chat.Nickname` is persisted across game starts, so this only runs once ever.  The complaint in #9728 is that the default for this doesn't change alongside the real player name.